src/components/tabs/Tabs.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import CSSModules from 'react-css-modules'
+import styles from './Tabs.css'
+const Tabs = React.createClass({
+  displayName: 'Tabs',
+  propTypes: {
+    selected: React.PropTypes.number,
+    children: React.PropTypes.oneOfType([
+      React.PropTypes.array,
+      React.PropTypes.element
+    ]).isRequired
+  },
+  getDefaultProps () {
+    return {
+      selected: 0
+    }
+  },
+  getInitialState () {
+    return {
+      selected: this.props.selected
+    }
+  },
+  handleClick (index, event) {
+    event.preventDefault()
+    this.setState({
+      selected: index
+    })
+  },
+  _renderTitles () {
+    function labels (child, index) {
+      let activeClass = (this.state.selected === index ? 'active' : '')
+      return (
+        <li key={index}>
+          <a href='#'
+            styleName={activeClass}
+            onClick={this.handleClick.bind(this, index)}>
+            {child.props.label}
+          </a>
+        </li>
+      )
+    }
+    return (
+      <ul styleName='tabs__labels'>
```

Comment:
How serious do we treat accessibility?

In perfect shape tabs are focusable, react to keyhandlers and have some `aria` stuff.

But that's probably the extra mile and out of scope.

https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel